frontend/demo/src/model/Channel.ts

```diff
@@ -0,0 +1,56 @@
+export interface Channel {
+  name: string;
+  source: string;
+  sourceChannelId: string;
+  connected: boolean;
+  imageUrl?: string;
+}
+
+export interface ChannelPayload {
+  id: string;
+  name: string;
+  image_url: string;
+  source: string;
+  source_channel_id: string;
+}
+
+export interface ExploreChannelRequestPayload {
+  source: string;
+  token: string;
+}
+
+export interface ConnectChannelRequestPayload {
+  source: string;
+  source_channel_id: string;
```

Comment:
shouldn't be all of this camelCase now?

Reply:
if we want to do it that way, I need another mapper into this. Right now I am confused what the generator will generate, what we have to do and who will write what.

Reply:
talked with chris. not longer confused :D